spec/main.tex

```diff
@@ -1123,28 +1141,41 @@ \subsubsection{Threshold ($t$-out-of-$n$) Presinging}
 
 \begin{description}
 
-\item[\bf Input.] Size of signing set $t$, identities of parties in signing set,
- index $i\in [t]$, index map\footnote{$S(i)$ is the index that $P_i$ had at key-generation time.} $S : [t] \to [n]$, key share $K_{S(i)}$, context separation string $\sid$, security level $L$, curve $\E$ with generator $G$.
+\item[\bf Input.] 
+  Size of signing set $t$,
+  identities of parties in signing set,
+  index $i\in [t]$,
+  index map\footnote{$S(i)$ is the index that $P_i$ had at key-generation time.} $S : [t] \to [n]$,
+  key share $K_{S(i)}$,
+  context separation string $\sid$,
+  security level $L$, curve $\E$ with generator $G$.
+
+  {\bf HD-wallets inputs:} additive $\text{shift} \in \Fq$ \footnote{Deriving additive shift is up to specific standard of HD-wallets, e.g. see \cite{bip32} or \cite{slip10}} ($\text{shift} = 0$ disables HD derivation)
 
     \item[\bf Setup.] The key share $K_{S(i)}$ contains $\threshold$, number of key holders~$n$, secret share $x'_{S(i)}$, parties' public shares $\vec X' = (X'_j)_{j \in [n]}$, a map $I : [n] \to \Fq \setminus \{0\}$, Paillier secret key $\sk_{S(i)}$, parties' Paillier keys $\vec N' = (N'_j)_{j \in [n]}$, and parties' auxiliary information $\vec R' = (s_j, t_j, \hat N)_{j \in [n]}$.
 
     \item[\bf Step 1.] Set $\sk_i = \sk_{S(i)}$ and 
-            $\vec R = (R'_{S(j)})_{j \in [n]}$. Then:
+                       $\vec R = (R'_{S(j)})_{j \in [n]}$. Then:
```

Comment:
Tab character or specific over-indentation?

Reply:
I leveled it with previous line, like we did in many other places